lasagne/layers/__init__.py

```diff
@@ -8,4 +8,5 @@
 from .shape import *
 from .merge import *
 from .normalization import *
+from .prelu import *
```

Comment:
Yeah, that would be confusing. What about `layers.elemwise`, `transfer` or `activation`?